src/components/Modals/Nfts/components/NftProperties.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Flex, useColorModeValue } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { RawText, Text } from 'components/Text'
+
+const exampleProperties = [
+  {
+    label: 'Mouth',
+    value: 'M2 bored unshaven pizza',
+  },
+  {
+    label: 'Earring',
+    value: 'M2 cross',
+  },
+  {
+    label: 'Clothes',
+    value: 'M2 bayc t black',
+  },
+  {
+    label: 'Fur',
+    value: 'M2 cheetah',
+  },
+  {
+    label: 'Background',
+    value: 'M2 blue',
+  },
+  {
+    label: 'Eyes',
+    value: 'M2 coins',
+  },
+  {
+    label: 'Hat',
+    value: 'M2 party hat 1',
+  },
+]
+export const NftProperties = () => {
```

Comment:
currently unconsumed but will hook this up to attribute data when we get opensea api key